docs/src/styles/docs/header.scss

```diff
@@ -2,15 +2,15 @@
   display: flex;
   position: sticky;
   top: 0;
-  width: 100vw;
+  width: 100%;
```

Comment:
This fixes a horizontal scrollbar bug. 100vw doesn't subtract for scollbars, so when a vertical scrollbar is present it's too wide and the page shows a dreaded horizontal scrollbar. In this case setting this to 100% of the parent container works great.

Reply:
🙌